bundle.md

```diff
@@ -1,39 +1,28 @@
-# Bundle
+# Filesystem Bundle
 
 ## Container Format
 
-This section defines a format for encoding a container as a *bundle* - a directory organized in a certain way, and containing all the necessary data and metadata for any compliant runtime to perform all standard operations against it.
+This section defines a format for encoding a container as a *filesystem bundle* - a set of files organized in a certain way, and containing all the necessary data and metadata for any compliant runtime to perform all standard operations against it.
 See also [OS X application bundles](http://en.wikipedia.org/wiki/Bundle_%28OS_X%29) for a similar use of the term *bundle*.
 
-The format does not define distribution.
-In other words, it only specifies how a container must be stored on a local filesystem, for consumption by a runtime.
-It does not specify how to transfer a container between computers, how to discover containers, or assign names or versions to them.
-Any distribution method capable of preserving the original layout of a container, as specified here, is considered compliant.
+The definition of a bundle is only concerned with how a container, and its configuration data, are stored on a local file system so that it can be consumed by a compliant runtime.
+Issues such as distribution, including how to transfer a container between runtimes, assigning names, versioning of bundle, or discovery of bundles are out of scope of this specification.
```

Comment:
I think what's there is more appropriate since all we can say is a reflection of the charter - and as of now its "out of scope" - we will never cover this w/o a charter change.  Using a word like "unspecified" could be interpreted as "we're choosing not to define it at this time but we could later", but that's not true w/o a charter change.

@vbatts : what's in or out of scope is still TBD. As much as I think the current scope of what's in this doc & PR says is too limited, I'm trying to go with what's agreeable for "in scope" for now and then will let the charter discussions change this doc later.